client/src/commons/Forms/PlacesTypesAndSubTypes/usePlacesTypesAndSubTypes.ts

```diff
@@ -28,6 +34,21 @@ const usePlacesTypesAndSubTypes = (parameters: usePlacesTypesAndSubTypesIncome)
         })
     };
 
+    const moveOtherValueLocationToLast = (placetypes:Array<{id:number,displayName:string}>) => {
+        let foundOtherValue = false;
+        //Checks if the 'other' value is not last already
+        if(placetypes[placetypes.length-1].displayName != OTHER){
+            for(let index = 0; index < placetypes.length && !foundOtherValue; index++){
```

Comment:
change to some

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/some

you can use it as a forEach with break

Reply:
Changed the function as Maya did